src/main/java/com/terraformersmc/cinderscapes/block/PyracinthBlock.java

```diff
@@ -29,4 +30,11 @@ public void randomDisplayTick(BlockState state, World world, BlockPos pos, Rando
             world.addParticle(ParticleTypes.FLAME, pos.getX() + 0.4f + random.nextFloat() * 0.2f, pos.getY() + 0.5f, pos.getZ() + 0.4f + random.nextFloat() * 0.2f, 0.0f, 0.05f, 0.0f);
         }
     }
+
+    @SuppressWarnings("deprecation")
+    @Override
+    public VoxelShape getOutlineShape(BlockState state, BlockView world, BlockPos pos, ShapeContext context) {
+        Vec3d vec3d = state.getModelOffset(world, pos);
+        return SHAPE_SUPPLIER.apply(state).offset(vec3d.x, vec3d.y, vec3d.z);
```

Comment:
We should just use `super(state, world, pos, context).offset(vec3d.x, vec3d.y, vec3d.z);` and make the supplier private in CinderscapesNetherPlantBlock.